### PR TITLE
Upgrade deb image to Stretch

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to upgrade our `web` image to extend from the `stretch` python image tag versus `jessie`.  Additionally, this PR upgrades some of our pinned python packages.
 
##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Testing
Built image and tested general functionality locally.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,48 +1,48 @@
 bs4
 ccxt
 cryptocompare==0.6
-cryptography==2.1.4
+cryptography==2.2.2
 django==2.0.4
-django-bootstrap3==8.2.3
+django-bootstrap3==9.1.0
```
Comment (thelostone-mc) [inline requirements/base.txt]: @mbeacom Thoughts on shifting to [django-bootstrap4](https://github.com/zostera/django-bootstrap4/) ?

Comment (mbeacom) [reply requirements/base.txt]: I'd love to... but I didn't want to make any **major** changes to the python packages in this PR.  The main thing here is the upgrade from `jessie` to `stretch` for LTS and upgraded system packages like `OpenSSL` jumping to `1.1`

We can definitely raise a PR following this to upgrade to `django-bootstrap4` - Thanks for looking out @thelostone-mc 😛 - If no one has objections, we could try it in this PR.  Looking to merge this following #668

Comment (thelostone-mc) [reply requirements/base.txt]: Let's give it a shot post #668 merge :D

Comment (mbeacom) [reply requirements/base.txt]: We can probably just rip out `django-bootstrap3` since it's unused throughout the codebase.

**Pull request merged**